examples/text_labels_and_annotations/shaded_table.py

```diff
@@ -0,0 +1,73 @@
+from matplotlib import pyplot as plt
```

Comment:
We are starting to switch to using sphinx gallery for generating the examples, see https://github.com/matplotlib/matplotlib/blob/master/examples/frontpage/contour.py for an example of what the header should look like